### DISCUSSION TITLE
Error in Decompose (Time Series)

### DISCUSSION THREAD

**Discussion by eeinnew** [teamalgoritma/community#131] [Q&A] [answered]
Hello, I am working with Time Series Forecasting, and when I run the Decompose chunk it shows error.
```
Error in decompose(jlh_ts) : time series has no or less than 2 periods
```
does it mean that I have no seasonality in my data?

**Comment by kevbow** [comment, marked as answer]
Hi @eeinnew, thanks for the question.

The error means that your data has no more than twice the length of frequency data. 
Decomposing trends need at least twice length of the data since it calculates CMA of the data using frequency as its order.
You can either try another length of frequency or add more data to your ts object.

I hope this answer can help you. If it does, please mark this as answer. Thank you!
